modules/base/src/components/Base.vue

```diff
@@ -125,73 +126,114 @@ export default {
           icon: "cog",
           loaded: false
         },
+        {
+          name: "Support",
+          url: "_support",
+          icon: "help",
+          loaded: false,
+          bottom: true
+        },
+        {
+          name: "About",
+          url: "_about",
+          icon: "info",
+          loaded: false,
+          bottom: true
+        },
         {
           name: "Dev Console",
           url: "/ui/modules/console/",
           icon: "terminal",
-          loaded: false
+          loaded: false,
+          bottom: true
         }
       ]
     };
   },
   computed: {
-    status() {
-      return this.op.records["core:status/status"];
+    versions() {
+      return this.statusDB.records["core:status/versions"];
+    },
+    topMenu() {
+      return this.uiModules.filter(value => {
+        return !value.bottom;
+      });
+    },
+    bottomMenu() {
+      return this.uiModules.filter(value => {
+        return value.bottom;
+      });
     }
   },
   methods: {
-    selectUIModule(index) {
-      this.uiModules[index].loaded = true;
-      this.activeModule = this.uiModules[index].url;
+    selectUIModule(url) {
+      for (const index in this.uiModules) {
+        if (this.uiModules[index].url == url) {
```

Comment:
```suggestion
        if (this.uiModules[index].url === url) {
```

In JS, always use strict equality (`===`) unless you really want a loose check (`==`).

Reply:
Ah. JS. 😏